### ISSUE TITLE
Plotting 

### ISSUE THREAD

**Issue by meechan1983** [0todd0000/spm1dmatlab#148]
Hi Todd

I am have a slight issue at the moment but i cannot figure it out. Below is the code I am using. I am looking to plot a vertical rectangle box/shaded area which covers hypothesis test output and the corresponding area on the CI plot lie the attached image?

I sincerely appreciate air help again

Regards 

David

% plot hypothesis test:
subplot(223)
spmi  = spm1d.stats.ttest_paired(variableA, variableB).inference(alpha);
spmi.clusters{:,:}
spmi.plot();

spmi.plot_threshold_label();
spmi.plot_p_values();

hold on;
%plot CI (possibility 3)
subplot(221)
ci2.plot()
xlim(myxlim)

<img width="482" alt="Screenshot 2021-06-02 at 10 19 08 PM" src="https://user-images.githubusercontent.com/83319591/120497116-b91a2d80-c3f0-11eb-8e79-6be36e615375.png">

**Comment by 0todd0000** [comment]
Try using MATLAB's [patch](https://www.mathworks.com/help/matlab/ref/patch.html) function.


```octave
%(0) Load data:
dataset = spm1d.data.uv1d.t1.Random();
% dataset = spm1d.data.uv1d.t1.SimulatedPataky2015a();
% dataset = spm1d.data.uv1d.t1.SimulatedPataky2015b();
[Y,mu]  = deal(dataset.Y, dataset.mu);



%(1) Conduct SPM analysis:
spm       = spm1d.stats.ttest(Y - mu);
spmi      = spm.inference(0.05, 'two_tailed',false);


%(2) Plot:
close all
spmi.plot();

c = spmi.clusters{1};
x1 = c.endpoints(1);
x2 = c.endpoints(2);

x = [x1 x2 x2 x1];
y = [0 0 5 5];

patch(x, y,'red')
```

**Comment by meechan1983** [comment]
Hi Todd

Many thanks for your help again.

Kind regards
David
Sent from my iPhone

On 10 Jun 2021, at 1:25 PM, Todd Pataky ***@***.***> wrote:

﻿

Closed #148<https://github.com/0todd0000/spm1dmatlab/issues/148>.
